Stratis.Bitcoin/BlockPulling/BlockPuller.cs

```diff
@@ -439,7 +347,7 @@ private void DistributeDownload(Dictionary<int, InventoryVector> vectors, BlockP
                 foreach (int blockHeight in blockHeightsToDownload)
                 {
                     InventoryVector inventoryVector = vectors[blockHeight];
-                    if (this.map.TryAdd(inventoryVector.Hash, peerBehavior))
+                    if (AssignDownloadTaskToPeer(peerBehavior, inventoryVector.Hash))
```

Comment:
erm, not sure about this one, i thought we use this. on fields and properties of the current instance, but this is method and MSVS gives me the hint that this. is missing (according to the rules we have on the project), but not on methods

i think calling methods should be without this.
right?